Dockerfile.dev

```diff
@@ -1,28 +1,4 @@
-FROM golang:1.11
-VOLUME /var/lib/openbazaar
-
-RUN wget https://www.python.org/ftp/python/3.6.0/Python-3.6.0.tgz && \
-    tar xvf Python-3.6.0.tgz && \
-    cd Python-3.6.0 && \
-    ./configure --enable-optimizations && \
-    make -j8
-RUN apt-get update && apt-get install -yq zlib1g-dev libssl-dev unzip
-RUN cd Python-3.6.0 && \
-    make altinstall && \
-    ln -s /usr/local/bin/python3.6 /usr/local/bin/python3
-
-COPY ./qa/requirements.txt ./requirements.txt
-
-RUN pip3.6 install --upgrade pip && \
-    pip3.6 install -r requirements.txt && \
-    wget https://bitcoin.org/bin/bitcoin-core-0.16.3/bitcoin-0.16.3-x86_64-linux-gnu.tar.gz && \
-    tar -xvzf bitcoin-0.16.3-x86_64-linux-gnu.tar.gz -C /opt
-
-RUN wget https://github.com/google/protobuf/releases/download/v3.6.0/protoc-3.6.0-linux-x86_64.zip && \
-    unzip ./protoc-3.6.0-linux-x86_64.zip -x readme.txt && \
-    mv ./include/* /usr/local/include/ && \
-    mv ./bin/protoc /usr/local/bin/ && \
-    rm -rf ./include ./bin
+FROM docker.dev.ob1.io/openbazaar-qa:0.10
```

Comment:
👍 very nice
![image](https://user-images.githubusercontent.com/45482/70827879-2e86de00-1db8-11ea-90b6-3fa27f9f3953.png)

Reply:
I wonder if anyone not authenticated against our registry will be able to pull this image.

Reply:
Not sure. @tyler-smith ?